company/team/README.md

```diff
@@ -0,0 +1,22 @@
+# Update yourself to Team Map
```

Comment:
Doesn't this repeat the instructions in https://github.com/sourcegraph/about/pull/1482/files#diff-258d0a09fc67cbd1883366d80fa02e1b?

Reply:
https://github.com/sourcegraph/about/pull/1488